### PR TITLE
start reading signal messages before session handler finishes

### PR DESCRIPTION
**Pull request by paulwe**

### CODE REVIEW

File: pkg/service/signal.go [hunk 1]
```diff
@@ -139,26 +139,34 @@ func (r *signalService) RelaySignal(stream psrpc.ServerStream[*rpc.RelaySignalRe
...
 	err = r.sessionHandler(ctx, livekit.RoomName(ss.RoomName), *pi, livekit.ConnectionID(ss.ConnectionId), reqChan, sink)
 	if err != nil {
 		l.Errorw("could not handle new participant", err)
 		return
 	}
 
-	err = routing.CopySignalStreamToMessageChannel[*rpc.RelaySignalResponse, *rpc.RelaySignalRequest](stream, reqChan, signalRequestMessageReader{}, r.config)
-	l.Infow("signal stream closed", "error", err)
-
+	stream.Hijack()
```
Comment (boks1971) [inline pkg/service/signal.go:169]: Just curious. What does Hijack do?

Comment (davidzhao) [inline pkg/service/signal.go:169]: what does Hijack do?

**Review by davidzhao — APPROVED** [review]

**Review by boks1971 — APPROVED** [review]

File: pkg/service/signal.go (hunk 1 above)
Comment (davidzhao) [reply pkg/service/signal.go:169]: hah jinx

Comment (paulwe) [reply pkg/service/signal.go:169]: normally when the stream handler function returns the stream is closed by psrpc - when a stream is hijacked it's up to the handler to close it manually

**Pull request merged**